### PR TITLE
[Docs] functions C-M - parameter and syntax fixes

### PR DESCRIPTION
**Pull request by seaniepie**
parameter and syntax fixes to the following functions:
codepointProperty, compound, decompress, libUrlMultipartFormAddPart,
localLoc, measureText and mobileBuildInfo

### CODE REVIEW

File: docs/dictionary/function/compound.lcdoc [hunk 1]
```diff
@@ -45,11 +44,10 @@ The formula for the value of an compound-interest-bearing account is
 The <compound> <function> calculates this value.
 
 The <numberOfPeriods> and the <interestRate> must use the same unit of
-time. For example, if the periods are months, the interest rate is the
+time. For example, if the periods are months, the <interest rate> is the
```
Comment (peter-b) [inline docs/dictionary/function/compound.lcdoc]: I think `<interest rate>` should be `<interestRate>` here.

Comment (seaniepie) [reply docs/dictionary/function/compound.lcdoc]: Well spotted. Missed that didn't I. Thanks @peter-b

**Comment by peter-b** [comment]
@livecode-vulcan review ok 4a709fb

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 4a709fb0fd2fa97e798e7f1d503043c5db95618c

**Comment by livecode-vulcan** [comment]
😞 test failure 4a709fb0fd2fa97e798e7f1d503043c5db95618c
- try-community-linux-x86: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/527)
- try-community-mac: [pending](https://vulcan.livecode.com/builders/try-community-mac/builds/532)
- try-community-linux-x86_64: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/527)
- try-community-emscripten: [failure](https://vulcan.livecode.com/builders/try-community-emscripten/builds/531)
- try-community-ios: [pending](https://vulcan.livecode.com/builders/try-community-ios/builds/536)
- try-community-win-x86: [pending](https://vulcan.livecode.com/builders/try-community-win-x86/builds/528)
- try-community-android-armv6: [pending](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/528)

**Comment by peter-b** [comment]
@livecode-vulcan retry 4a709fb

**Comment by livecode-vulcan** [comment]
😎 test success 4a709fb0fd2fa97e798e7f1d503043c5db95618c
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/533)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/538)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/533)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/537)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/542)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/534)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/534)

**Pull request merged**